packages/react-core/src/demos/ComposableMenu/examples/ComposableActionsMenu.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { MenuToggle, Menu, MenuList, MenuItem, MenuGroup, MenuItemAction, Popper } from '@patternfly/react-core';
+import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
+import ClipboardIcon from '@patternfly/react-icons/dist/esm/icons/clipboard-icon';
+import CodeBranchIcon from '@patternfly/react-icons/dist/esm/icons/code-branch-icon';
+import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+
+export const ComposableActionsMenu: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selectedItems, setSelectedItems] = React.useState([]);
```

Comment:
I think since these are IDs from what it looks like we can call this `selectedItemIds` instead, and we can add a type to the useState of `number[]`.

Reply:
I agree with adding a type to `React.useState([])`. Adding boolean to `React.useState(false)` seems redundant, considering this type is inferred